.github/workflows/publish-ui.yml

```diff
@@ -0,0 +1,79 @@
+name: "Publish @leav/ui @next package to npmjs on push on develop"
+
+on:
+  push:
+    branches:
+      - develop
+      - chore/publish-ui-next # TO DELETE
```

Comment:
**question:** Est-ce qu'il faut supprime ce `#TO DELETE` ?

Reply:
C'est pour tester sans avoir à push sur develop. Je cleanerai le fichier juste avant de merger.